cmake-naming.md

```diff
@@ -0,0 +1,36 @@
+## CMake targets for Mbed "targets"
+
+### targets.json naming scheme
+
+- `MCU_STM32L4` - base target
+- `DISCO_L4R9I` - final target - a user board or could be custom target, inherits from MCU_  target
+
+The targets.json properties use capital letters with underscore but not the our config. Probably to distinguish these from other regular directories (the initial Mbed SDK had it this way).
```

Comment:
> The targets.json properties use capital letters with underscore but not the our config.

Can you give an example? I am not sure what you mean here.

Reply:
```
        "overrides": {
            "tickless-from-us-ticker": true,
            "init-us-ticker-at-boot": true
        },
```